### DISCUSSION TITLE
copy/paste support

### DISCUSSION THREAD

**Discussion by MarcinBudek** [vvmgev/Inssman#55] [Request Feature]
it will be helpful when user can paste value in "modify-response" instead of typing it

**Comment by Mishasama** [comment]
You can paste it as plain text, but not in rich format.

**Reply by Bi1iOne** [reply]
no way

**Reply by maxpowernz** [reply]
I cant even paste plain text.

**Reply by shironaomi** [reply]
> I cant even paste plain text.

Me neither

**Reply by Mishasama** [reply]
> I cant even paste plain text.

Note that the browser's default paste behaviour is in rich text.

**Reply by ihimv** [reply]
I tried pasting from notepad but that also didn't work. Luckily, I found a hack, use `ctrl + shift + v` to paste instead of just ctrl + v. The dev needs to update the monaco editor to latest version and this issue will be resolved.
